test/utils/SafeCast.test.js

```diff
@@ -42,5 +42,44 @@ describe('SafeCast', async () => {
     });
   }
 
+  describe('toUint256', () => {
+    const maxInt256 = new BN('1').shln(255).notn(256);
+    const minInt256 = new BN('1').shln(255);
+    const maxUint256 = new BN('0').notn(256);
+
+    it('downcasts 0', async function () {
```

Comment:
```suggestion
    it('casts 0', async function () {
```

We're not really downcasting here, so these should just read 'casts' :stuck_out_tongue:

Reply:
Added in the next commit!